.travis.yml

```diff
@@ -13,11 +13,14 @@ matrix:
   include:
     - php: 7.1
       env: PHPSTAN=1
+           CODING_STANDARD=1
 
 before_script:
   - composer --prefer-source install
   - if [[ $PHPSTAN = 1 ]]; then composer require --dev phpstan/phpstan:^0.7; fi
+  - if [[ $CODING_STANDARD = 1 ]]; then composer require --dev symplify/easy-coding-standard:^2.2; fi
```

Comment:
I don't see a reason to use just one tool, since they have great ecosystem that work best together, see `nette/coding-standard.neon`: https://github.com/nette/coding-standard/blob/master/coding-standard-php56.neon

Instead of writing own sniff/fixer that is missing in the other tool (and burn out in the process), you can just use it.

You can always fallback to just one tool if this causes any troubles replacing this line.